etc/profile-m-z/neochat.profile

```diff
@@ -0,0 +1,75 @@
+# Firejail profile for neochat
+# Description: Matrix Client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include neochat.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/KDE/neochat
+noblacklist ${HOME}/.config/KDE/neochat
+noblacklist ${HOME}/.config/neochatrc
+noblacklist ${HOME}/.config/neochat.notifyrc
+noblacklist ${HOME}/.local/share/KDE/neochat
+noblacklist ${DOWNLOADS}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/KDE/neochat
+mkdir ${HOME}/.config/KDE/neochat.conf
+mkdir ${HOME}/.local/share/KDE/neochat
+mkfile ${HOME}/.config/neochatrc
+mkfile ${HOME}/.config/neochat.notifyrc
+
+whitelist ${HOME}/.cache/KDE/neochat
+whitelist ${HOME}/.config/KDE/neochat.conf
+whitelist ${HOME}/.config/neochatrc
+whitelist ${HOME}/.local/share/KDE/neochat
+whitelist ${HOME}/.config/neochat.notifyrc
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-bin neochat
```

Comment:
`private-cache`?

Reply:
`private-cache` makes the application loading visibly slow. When I was testing the profile, I thought it wasn't working (due to the very slow loading), which is why I did not add that option. Should I add it?

Reply:
then, no